middleware/gateway/source/manager/main.cpp

```diff
@@ -54,6 +54,10 @@ namespace casual
                return transform::state( casual::domain::configuration::fetch().gateway);
             }
 
+#if CASUAL_PROTOCOL_VERSION == 1002
```

Comment:
What is happening here?

Reply:
Ahh, leftovers from playing with compiler defines.